esp/clients/roxiecontrol.hpp

```diff
@@ -21,11 +21,19 @@
 #include "jlib.hpp"
 #include "jsocket.hpp"
 #include "jptree.hpp"
-#include "roxiecontrol.hpp"
+#include "jsmartsock.hpp"
+
+const unsigned ROXIECONNECTIONTIMEOUT = 1000;   //1 second
+const unsigned ROXIECONTROLQUERYTIMEOUT = 3000; //3 second
```

Comment:
What is the difference between this and the next timeout?

Reply:
I may have misinterpreted it slightly.  These were existing values that I moved, and I thought they were general purpose, (just badly named), but the `ROXIECONTROLQUERIESTIMEOUT ` one at least looks more specific to `"<control:queries>"`.  I'll add new ones where appropriate to avoid confusion.

Reply:
Adding a value specific to `control:getQueryXrefInfo` and assigning the value 5000 which is what's been used for that query in the past.